tools/myst/enc/clock.c

```diff
@@ -162,3 +181,23 @@ long myst_tcall_clock_settime(clockid_t clk_id, struct timespec* tp)
     // Clocks other than CLOCK_REALTIME are not settable
     return -EINVAL;
 }
+
+/**
+ * memcpy for 8-byte aligned count.
+ *
+ * @param[out] dest Memory where the content is to be copied.
+ * @param[in] src A pointer to the source of data to be copied.
+ * @param[in] count The number of bytes to be copied.
+ *
+ */
+OE_ALWAYS_INLINE
+void oe_memcpy_aligned(void* dest, const void* src, size_t count)
```

Comment:
Given that we can use OE header here, we can just update the OE version as part of the PR and then this API should be available.

To update the OE, change the commit hash at
https://github.com/deislabs/mystikos/blob/ddccafad171c9c24accb2a4b7605e937c319909a/third_party/openenclave/Makefile#L9
to `d8e1004945f255a7b7b62f4cb3bb43e0e3a83c0a` (the latest OE master)